File: BOJ/class2_bronze.js
// https://www.acmicpc.net/problem/2798 [블랙잭] - 브루트 포스(brute force) 알고리즘 : 무식한 완전탐색 알고리즘!
// https://tesseractjh.tistory.com/4(참고)

// const readline = require('readline');
// const rl = readline.createInterface({
//   input: process.stdin,
//   output: process.stdout
// });

// let input = [];
// rl.on('line', function (line) {
//   input.push(parseInt(line));
// }).on('close', function () {
//   let N = parseInt(input[0]);
//   let M = parseInt(input[1]);
//   let sum = 0;
//   let cards = input.slice(2); //2번째 줄부터 입력받은 숫자들을 배열 형태로 저장.
  
//   for(let i = 0; i < N-2; i++){ 
//       for(let j = i+1; j < N-1; j++){
//           for(let k = j+1; k < N; k++){
//               if(cards[i]+cards[j]+cards[k] <= M && cards[i]+cards[j]+cards[k] > sum){
//                       sum = cards[i]+cards[j]+cards[k];
//               }
//           }
//       }
//   }
  
//   console.log(sum);

// process.exit();
// });

//https://www.acmicpc.net/problem/4153 [직각삼각형]

const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

let input = [];

rl.on('line', function (line) {
    if(line == '0 0 0') rl.close();
    else input.push(line);
  })
  .on('close', function () {
    for (let i = 0; i < input.length; i++) {
      let nums = input[i].split(" ");
      nums.sort((a,b) => a - b);
      let num1 = Number(nums[0]);
      let num2 = Number(nums[1]);
      let num3 = Number(nums[2]);
    
      if ((num1 * num1) + (num2 * num2) == num3 * num3) console.log("right");
      else console.log("wrong");
    }
    process.exit();
});

File: BOJ/class2_bronze.py
# https://www.acmicpc.net/problem/4153 [직각삼각형]
# import sys
#
# while True:
#     triangles = list(map(int, sys.stdin.readline().split())) #리스트에 넣은 후
#     triangles.sort() #소팅한다. sort()는 본체의 list를 정렬해서 반환하고, sorted는 정렬한 새로운 list를 반환한다.
#     if triangles[0] == 0:
#         break
#     elif ((triangles[0]**2) + (triangles[1]**2)) == (triangles[2]**2): #소팅하면 제일 큰게 마지막 idx를 가진다.
#         print("right")
#     else:
#         print("wrong")

# https://www.acmicpc.net/problem/1920 [수 찾기]
# 리스트의 in연산자를 통한 포함 여부의 시간 복잡도는 O(N)
# 이분 탐색의 시간 복잡도는 O(logN)
# Set과 Dictionary의 in연산을 통한 포함 여부 확인의 시간 복잡도는 O(1)
# 해당 집합에 포함되는지 여부만 확인하면 되므로 Set 자료형을 사용해도 충분
# import sys
#
# nothing = int(input())
# base_list = set(map(int, sys.stdin.readline().split()))
# num = int(input())
# target_list = list(map(int, sys.stdin.readline().split()))
#
# for i in target_list:
#     if i in base_list:
#         print('1')
#     else:
#         print('0')

# https://www.acmicpc.net/problem/2798 [블랙잭]
# import sys
#
# N,M = map(int,sys.stdin.readline().split())
# cards = list(map(int, sys.stdin.readline().split()))
# sum = 0
#
# for i in range(N-2):
#     for j in range(i+1, N-1):
#         for k in range(j+1, N):
#             if cards[i]+cards[j]+cards[k] <= M and cards[i]+cards[j]+cards[k] > sum:
#                 sum = cards[i]+cards[j]+cards[k]
# print(sum)

# https://www.acmicpc.net/problem/11050 [이항 계수]
# import sys

# N,K = map(int,sys.stdin.readline().split())
# top = 1
# bottom = 1
# minus = 1
# for n in range(N,0,-1):
#     top *= n
# for k in range(K,0,-1):
#     bottom *= k
# for m in range(N-K,0,-1):
#     minus *= m

# print(int(top / (bottom * minus)))

# https://www.acmicpc.net/problem/10250 [ACM 호텔]

import sys
n = int(input())

for i in range(n):
    H, W, N  = map(int,sys.stdin.readline().split())
    num = N / H + 1
    floor = N % H
    if N % H == 0:  # h의 배수이면,
        num = N/H
        floor = H
    print(int(floor*100+num))


